### PR TITLE
Rollup branch for updating SEO for HCP Terraform

### PR DESCRIPTION
**Pull request by rkoron007**
### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. HCP Terraform ).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added to `website/redirects.js` for moved, renamed, or deleted pages.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Review by ritsok — APPROVED** [review]

File: website/docs/cloud-docs/policy-enforcement/define-policies/custom-sentinel.mdx [hunk 1]
```diff
@@ -1,11 +1,9 @@
 ---
-page_title: Define custom Sentinel policies
-description: >-
-  Learn how to use Sentinel policy language to create custom policies, including
-  imports to define rules and useful functions.
+page_title: Define policies with Sentinel in HCP Terraform
```
Comment (ritsok) [inline website/docs/cloud-docs/policy-enforcement/define-policies/custom-sentinel.mdx:2]:
```suggestion
page_title: Define Sentinel policies in HCP Terraform
```

File: website/docs/cloud-docs/policy-enforcement/define-policies/custom-sentinel.mdx [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
+description: Learn how to use the Sentinel policy language to create policies in HCP Terraform.
 ---
 
-# Define custom Sentinel policies 
+# Define policies with Sentinel in HCP Terraform
```
Comment (ritsok) [inline website/docs/cloud-docs/policy-enforcement/define-policies/custom-sentinel.mdx:6]:
```suggestion
# Define Sentinel policies in HCP Terraform
```

File: website/docs/cloud-docs/policy-enforcement/test-sentinel.mdx [hunk 3]
```diff
@@ -1,11 +1,10 @@
 ---
-page_title: Mocking Terraform Sentinel Data - Sentinel - HCP Terraform
+page_title: Mock Terraform Sentinel data with HCP Terraform
```
Comment (ritsok) [inline website/docs/cloud-docs/policy-enforcement/test-sentinel.mdx:2]:
```suggestion
page_title: Generate mock Sentinel data with HCP Terraform
```

File: website/docs/cloud-docs/recommended-practices/part3.mdx [hunk 4]
```diff
@@ -1,13 +1,12 @@
 ---
 page_title: >-
-  Part 3: How to Evolve Your Provisioning Practices - Terraform Recommended
-  Practices
+  Part 3: Learn how to evolve Your provisioning practices
```
Comment (ritsok) [inline website/docs/cloud-docs/recommended-practices/part3.mdx:3]:
```suggestion
  Part 3: Evolve your provisioning practices
```

File: website/docs/cloud-docs/recommended-practices/part3.mdx [hunk 5]
```diff
@@ -1,13 +1,12 @@
...
 tfc_only: true
 description: >-
   Use Terraform to support a collaborative infrastructure as code workflow. Learn how to evolve your provisioning practices.
 ---
 
-# Part 3: How to Evolve Your Provisioning Practices
+# Part 3: Learn how to evolve Your provisioning practices
```
Comment (ritsok) [inline website/docs/cloud-docs/recommended-practices/part3.mdx:9]:
```suggestion
# Part 3: Evolve your provisioning practices
```

File: website/docs/cloud-docs/run/states.mdx [hunk 6]
```diff
@@ -1,11 +1,10 @@
 ---
-page_title: Run States and Stages - Runs - HCP Terraform
+page_title: Run states and stages in HCP Terraform
 description: >-
-  Learn what happens during each stage of a run: pending, plan, cost estimation,
-  policy check, apply, and complete.
+  Learn the run stages that HCP Terraform goes through to perform Terraform operations. Understanding run stages and their states can help you follow a run's flow.
```
Comment (ritsok) [inline website/docs/cloud-docs/run/states.mdx:4]:
```suggestion
  Learn the run stages of Terraform operations. Understanding run stages and their states can help you follow a run's progress.
```

File: website/docs/cloud-docs/workspaces/index.mdx [hunk 7]
```diff
@@ -1,8 +1,7 @@
 ---
-page_title: Workspaces - HCP Terraform
+page_title: Workspaces in HCP Terraform
```
Comment (ritsok) [inline website/docs/cloud-docs/workspaces/index.mdx:2]:
```suggestion
page_title: HCP Terraform workspaces
```

File: website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx [hunk 8]
```diff
@@ -1,10 +1,10 @@
 ---
-page_title: SSH Keys for Cloning Modules - Workspaces - HCP Terraform
+page_title: Use SSH Keys for cloning modules in HCP Terraform
```
Comment (ritsok) [inline website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx:2]:
```suggestion
page_title: Use SSH Keys to clone modules in HCP Terraform
```

File: website/docs/registry/providers/docs.mdx [hunk 9]
```diff
@@ -1,11 +1,10 @@
 ---
-page_title: Terraform Registry - Provider Documentation
+page_title: Provider documentation in the Terraform registry
```
Comment (ritsok) [inline website/docs/registry/providers/docs.mdx:2]:
```suggestion
page_title: Publishing provider documentation in the Terraform registry
```

File: website/docs/registry/policy-libraries/publishing.mdx [hunk 10]
```diff
@@ -1,10 +1,10 @@
 ---
-page_title: Terraform Registry - Publishing Policy Libraries
+page_title: Publish policy libraries in the Terraform registry
```
Comment (ritsok) [inline website/docs/registry/policy-libraries/publishing.mdx:2]:
```suggestion
page_title: Publish policy libraries to the Terraform registry
```

File: website/docs/registry/modules/publish.mdx [hunk 11]
```diff
@@ -1,9 +1,9 @@
 ---
-page_title: Terraform Registry - Publishing Modules
-description: Anyone can publish and share modules on the Terraform Registry.
+page_title: Publish modules in the Terraform registry
```
Comment (ritsok) [inline website/docs/registry/modules/publish.mdx:2]:
```suggestion
page_title: Publish modules to the Terraform registry
```

File: website/docs/registry/providers/publishing.mdx [hunk 12]
```diff
@@ -1,9 +1,9 @@
 ---
-page_title: Terraform Registry - Publishing Providers
-description: Learn how to release a provider and publish it to the Terraform Registry.
+page_title: Publish providers in the Terraform registry
```
Comment (ritsok) [inline website/docs/registry/providers/publishing.mdx:2]:
```suggestion
page_title: Publish providers to the Terraform registry
```

File: website/docs/registry/providers/publishing.mdx [hunk 13]
```diff
@@ -1,9 +1,9 @@
...
+description: Learn how to create, document, release, and publish a provider in the Terraform registry.
```
Comment (ritsok) [inline website/docs/registry/providers/publishing.mdx:3]:
```suggestion
description: Learn how to create, document, release, and publish a provider to the Terraform registry.
```

File: website/docs/cloud-docs/migrate/tf-migrate/reference/execute.mdx [hunk 14]
```diff
@@ -1,10 +1,10 @@
 ---
-page_title: tf-migrate execute - Terraform migrate - HCP Terraform
+page_title: tf-migrate execute reference
 description: >-
-  Run a prepared migration.
+  The tf-migrate execute command runs a prepared migration to migrate locally existing state to HCP Terraform or Terraform Enterprise.
```
Comment (ritsok) [inline website/docs/cloud-docs/migrate/tf-migrate/reference/execute.mdx:4]:
```suggestion
  The `tf-migrate execute` command runs a prepared migration to migrate locally existing state to HCP Terraform or Terraform Enterprise.
```

File: website/docs/cloud-docs/migrate/tf-migrate/reference/execute.mdx [hunk 15]
```diff
@@ -1,10 +1,10 @@
...
 ---
 
-# tf-migrate execute
+# tf-migrate execute reference
```
Comment (ritsok) [inline website/docs/cloud-docs/migrate/tf-migrate/reference/execute.mdx:7]:
```suggestion
# `tf-migrate execute` reference
```

File: website/docs/cloud-docs/migrate/tf-migrate/reference/prepare.mdx [hunk 16]
```diff
@@ -1,10 +1,10 @@
 ---
-page_title: tf-migrate prepare - Terraform migrate - HCP Terraform
+page_title: tf-migrate prepare reference
 description: >-
-  Gather information and create a plan to migrate your Terraform Community Edition state.
+  The tf-migrate prepare command gathers information and creates a plan to migrate your Terraform Community Edition state.
```
Comment (ritsok) [inline website/docs/cloud-docs/migrate/tf-migrate/reference/prepare.mdx:4]:
```suggestion
  The `tf-migrate prepare` command gathers information and creates a plan to migrate your Terraform Community Edition state.
```

File: website/docs/cloud-docs/migrate/tf-migrate/reference/prepare.mdx [hunk 17]
```diff
@@ -1,10 +1,10 @@
...
 ---
 
-# tf-migrate prepare
+# tf-migrate prepare reference
```
Comment (ritsok) [inline website/docs/cloud-docs/migrate/tf-migrate/reference/prepare.mdx:7]:
```suggestion
# `tf-migrate prepare` reference
```

File: website/docs/cloud-docs/workspaces/settings/run-tasks.mdx [hunk 18]
```diff
@@ -1,12 +1,12 @@
 ---
-page_title: Run Tasks - Workspaces - HCP Terraform
+page_title: Run tasks in HCP Terraform
```
Comment (ritsok) [inline website/docs/cloud-docs/workspaces/settings/run-tasks.mdx:2]:
```suggestion
page_title: HCP Terraform run tasks
```

File: website/docs/cloud-docs/workspaces/settings/run-triggers.mdx [hunk 19]
```diff
@@ -1,10 +1,10 @@
 ---
-page_title: Run Triggers - Workspaces - HCP Terraform
+page_title: Run triggers in HCP Terraform
```
Comment (ritsok) [inline website/docs/cloud-docs/workspaces/settings/run-triggers.mdx:2]:
```suggestion
page_title: HCP Terraform run triggers 
```

File: website/docs/cloud-docs/workspaces/health.mdx [hunk 20]
```diff
@@ -1,7 +1,7 @@
 ---
-page_title: Health - HCP Terraform
+page_title: Health in HCP Terraform
```
Comment (ritsok) [inline website/docs/cloud-docs/workspaces/health.mdx:2]:
```suggestion
page_title: Health assessments in HCP Terraform
```

or Workspace health?

File: website/docs/docs/index.mdx [hunk 21]
```diff
@@ -1,12 +1,11 @@
 ---
-page_title: Documentation
+page_title: Terraform documentation
```
Comment (ritsok) [inline website/docs/docs/index.mdx:2]:
```suggestion
page_title: Terraform overview
```

Just a thought?

File: website/docs/docs/index.mdx [hunk 22]
```diff
@@ -1,12 +1,11 @@
...
 description: >-
-  Documentation for Terraform, including Terraform CLI, HCP Terraform, and
-  Terraform Enterprise.
+  Terraform enables you to provision infrastructure safely and efficiently. This documentation is for Terraform's ecosystem, including the Terraform CLI, HCP Terraform, and Terraform Enterprise.
```
Comment (ritsok) [inline website/docs/docs/index.mdx:4]:
```suggestion
  Terraform enables you to provision infrastructure safely and efficiently. Review Terraform's ecosystem and workflows, including the Terraform CLI, HCP Terraform, and Terraform Enterprise.
```

File: website/docs/docs/index.mdx [hunk 23]
```diff
@@ -1,12 +1,11 @@
...
 show_notification: false
 ---
 
-# Terraform Documentation
+# Terraform documentation
```
Comment (ritsok) [inline website/docs/docs/index.mdx:8]:
```suggestion
# Terraform overview
```

**Review by ritsok — APPROVED** [review]
thank you!

File: website/docs/cloud-docs/api-docs/organization-tags.mdx [hunk 24]
```diff
@@ -1,7 +1,7 @@
 ---
-page_title: Organization Tags - API Docs - HCP Terraform
+page_title: Organization tags API reference for HCP Terraform
```
Comment (BrianMMcClain) [inline website/docs/cloud-docs/api-docs/organization-tags.mdx:2]:
```suggestion
page_title: /tags API reference for HCP Terraform
```

**Pull request merged**